environments/skeleton/{{cookiecutter.environment}}/terraform/nodes.tf

```diff
@@ -76,6 +76,9 @@ resource "openstack_compute_instance_v2" "control" {
 
   metadata = {
     environment_root = var.environment_root
+    k3s_token = var.k3s_token
+    k3s_server = "${var.cluster_name}-control"
```

Comment:
Do we actually need this for the server??

Reply:
actually, we should be able to ditch the agent type key/value and just run an agent if the server is specified?